code/src/index.css

```diff
@@ -1,13 +1,169 @@
-body {
+/* ===== GLOBAL ATRUBUTES ===== */
+
+* {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  padding: 0;
+}
+
+body {
+  font-family: "Arial" ,"Helvetica Neue", sans-serif;
+  background-color: rgb(20, 20, 20);
+  color: #f1f1f1;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+h1 {
+  font-size: 30px;
+  margin: 10px auto;
+}
+
+header {
+  margin: 20px 23px;
+  border-bottom:2px solid  rgb(70, 70, 70);
+}
+
+a {
+  color: #f1f1f1;
+  text-decoration: none;
+}
+
+p {
+  font-size: 14px;
+}
+
+/* ===== MAIN CARD & OBJECTS ===== */
+
+.card {
+  position: relative;
+  margin: 10px;
+  padding: 20px;
+  width: 80%;
+  height: auto;
+  border-radius: 10px;
+  place-self: center;
+}
+
+.img-container {
+  position: relative;
+}
+
+.album-img {
+  opacity: 1;
+  display: block;
+  width: 100%;
+  height: auto;
+  transition: .5s ease;
+  backface-visibility: hidden;
+}
+
+.img-container:hover .album-img {
+  opacity: .3;
+}
+
+.img-container:hover .iconsContainer {
+  opacity: 1;
+}
+
+.artistLink{
+  font-size: 14px;
+}
+
+.albumTitle{
+  margin: 10px auto;
+  font-weight: bold;
+}
+
+/* ===== ICONS ===== */
+
+.iconsContainer {
+  position: absolute;
+  opacity: 0;
+  width: 80%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
 }
+
+.icons {
+  width: 50px;
+  filter: brightness(0) invert(1);
```

Comment:
We will definitely look a bit more at this filter function since it is new for us.